app/lib/three_scale/invitation_email_delivery_observer.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class ThreeScale::InvitationEmailDeliveryObserver
+  def self.delivered_email(message)
+    binding.pry
+    Invitation.find_by(email: message.to)
```

Comment:
Can an invitation to the same email be done for different accounts? If so, you should search by the account as well.

Reply:
You mean 2 accounts inviting the same email? Makes sense.

Reply:
yup 😄